packages/ui-components/src/components/utils.ts

```diff
@@ -1,10 +1,11 @@
+import { isNil } from 'lodash-es';
 import { DEFAULT_CONFIG } from '../globals/config';
 import { UIComponentsConfig } from '../types';
 
 export const getConfig = (): UIComponentsConfig => {
 	if (typeof window !== 'undefined') {
 		const KvUiComponents = window.KvUiComponents;
-		if (KvUiComponents && KvUiComponents.config) {
+		if (!isNil(KvUiComponents) && !isNil(KvUiComponents.config)) {
```

Comment:
is there any specific reason why we're changing the boolean expr to use `isNil()`?

lodash's `isNil` method compares a given value with the abstract comparison algorithm (a.k.a loose equality) to check for `undefined` or `null` (both falsy values), is there any other falsy value these can assume, or is this just to fix some warning in the linter? If so, can you write it here? other than that LGTM 💪

Reply:
I'm using the `IsNil()` from loadash exactly because it checks both equalities. But yes it's just to fix some warnings in linter 👌

Reply:
:shipit: